examples/custom-routes-rewrites/next.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  async rewrites() {
+    return [
+      {
+        source: '/about',
+        destination: '/pretty-long-file-name-for-an-about-page',
```

Comment:
Let's do something more simple here, like making `/team` go to `/about` 👍

Reply:
Let's also add another one: `/about-us` -> `/about`, and links to them in the jsx of the index page

Reply:
For another use case, lets add something like what we have in docs:

> To match a wildcard path you can use `*` after a parameter, for example `/blog/:slug*` will match `/blog/a/b/c/d/hello-world`:

```js
module.exports = {
  async rewrites() {
    return [
      {
        source: '/blog/:slug*',
        destination: '/news/:slug*', // Matched parameters can be used in the destination
      },
    ]
  },
}
```

so, have a dynamic route for `/news/:slug` (should be `/news/[...slug]`) that renders the query path like in the about page 👍

Reply:
Actually, let's make sure that the example matches the examples we have in the documentation page, and to also include:

- rewriting to an external URL
- basePath support

Don't include the one for `Incremental adoption of Next.js` as we already have an example for it.

Reply:
Sure. So you want all of these use cases in a single example right? @lfades

Reply:
@robintom All in this example 👍